src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -18,6 +18,11 @@ public static class SizeAndLocation
 
    public static class Captions
    {
+      public static string Transporter = "Transporter";
```

Comment:
Why are those guys static strign where everything under is readonly string? Should we make this consistent?

Reply:
done